Main.java

```diff
@@ -7,8 +7,7 @@ public static void main(String[] args) {
     GrassPokemon chicorita = new GrassPokemon("Chicorita", 300, 34, "Grass", 55, 200,"Guatamala");
     WaterPokemon wailmer = new WaterPokemon("Wailmer", 600, 41,"Water", 300, 300, true);
     FirePokemon torkoal = new FirePokemon("Torkoal", 150, 20,"fire", 200, "Black",false);
-    PsychicPokemon gardevoir = new PsychicPokemon("Gardevoir", 400,50, "Psychic/Fairy", 200,500,600);
-
+    PsychicPokemon gardevoir = new PsychicPokemon("Gardevoir", 400,50, "Psychic/Fairy", 200,500, 600);
```

Comment:
Deze regel is het enige verschil met de main-branch. Let op het stappenplan van git: 

- Clone het project van de novihogeschool github
- Maak een nieuwe repo op je eigen github
- git remote remove origin
- git remote add origin [je eigen repo]
- git push origin main
- git checkout -b [nieuwe branch naam, bijvoorbeeld "huiswerk"]
- Daarna kun je je huiswerk gaan maken op deze nieuwe branch.
- Vergeet niet regelmatig `git add .` en `git commit -m "message"` te doen.
- Als je klaar bent kun je al je commits in 1 keer pushen en kun je een Pull Request maken waar al je uitwerkingen op te zien zijn.